storey/table.py

```diff
@@ -1064,6 +1079,12 @@ def __init__(self, max_value=None, set_data=None, set_time=None):
     def aggregate(self, time, value):
         self._set_value(self.value + 1)
 
+    def aggregate_argument(self, time, argument):
+        self._set_value(self.value + argument)
+
+    def aggregate_lua_script(self, vl1, vl2):
+        return '{}+{}'.format(vl1, vl2)
```

Comment:
Hm, shouldn't this be
```suggestion
        return f'{vl1}+1'
```
(changed the style, but I'm referring to the fact that I expected a +1 here)

If I'm right, why didn't this fail any of the aggregation tests...?

Reply:
Not always. On aggregation I do count1+count2 and not count1+1